src/scripts/parseCompartmentDefinition.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from "fs";
+import * as path from "path";
+
+const compartmentDefPath = path.resolve(
+  path.join(__dirname, "../fhir/compartmentdefinition-patient.json")
+);
+const attrOutputPath = path.resolve(
+  path.join(__dirname, "../data/patient-attribute-paths.json")
+);
+const paramOutputPath = path.resolve(
+  path.join(__dirname, "../data/patient-search-parameters.json")
+);
+const jsonStr = fs.readFileSync(compartmentDefPath, "utf8");
+const { getSearchParameters } = require("@projecttacoma/node-fhir-server-core");
```

Comment:
I wasn't able to run this. I could be doing something wrong, but I got "Error: Cannot find module '@projecttacoma/node-fhir-server-core'". Does it need to be added as a dependency?

Reply:
Sorry! This must have gotten dropped from the package.json at some point.